### PR TITLE
Working Group info: add Forum

### PR DESCRIPTION
**Pull request by Andersson007**
Relates to https://forum.ansible.com/t/teams-community-building-blocks/182

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +46,7 @@ This repository serves as a space for the Ansible Community Working Group to man
 
 ## Speak to us
 
+* On the [Forum](https://forum.ansible.com/categories) in a suitable category
```
Comment (cybette) [inline README.md:49]: s/Forum/Ansible Community Forum/

since it's the first mention

Comment (cybette) [reply README.md:49]: ignore this comment, it was suppose to address the first mention of Forum :P

File: README.md [hunk 2]
```diff
@@ -19,17 +19,21 @@ This repository serves as a space for the Ansible Community Working Group to man
 
 ## Infrastructure and processes we build and/or support
 
+* [Forum](https://forum.ansible.com/): Official Ansible Discource forum.
```
[the remark this answers is not in the document]
Comment (cybette) [reply README.md:22]: or at least `Community Forum`

File: README.md [hunk 3]
```diff
@@ -19,17 +19,21 @@ This repository serves as a space for the Ansible Community Working Group to man
...
 * [Meetings](https://github.com/ansible/community/tree/main/meetings): Matrix/IRC meeting-related infrastructure, including the meeting bot and space for logs; organization of Ansible Contributor Summits.
 * [Bullhorn](https://github.com/ansible/community/wiki/News#the-bullhorn): Ansible contributor newsletter.
 * [Collection Bot](https://github.com/ansible-community/collection_bot): Automation of GitHub issue/pull request processing for ansible collections.
 
 ## Groups
 
-* [All Ansible Working Groups](https://github.com/ansible/community/wiki)
+* Ansible Working Groups are now present in two forms:
+  * [On the Forum](https://forum.ansible.com/g) (recommended)
+  * [As wiki pages](https://github.com/ansible/community/wiki) in this repository
```
Comment (GregSutcliffe) [inline README.md:31]: Do we want to add "deprecated" here? Our discussion was to create *new* groups via the forum and move the old ones over in time, so lets try not to create new ones here?

**Review by GregSutcliffe — APPROVED** [review]
Looks great - I'm resisting the temptation to change more of this, one thing at a time ;)

Minor nitpick about the old groups, but would be happy with it as is.

File: README.md (hunk 3 above)
Comment (cybette) [reply README.md:31]: +1 for adding "will be deprecated"

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Andersson007) [reply README.md]: SGTM, done, thanks!

File: README.md (hunk 3 above)
Comment (Andersson007) [reply README.md]: yeah, let's be clear;) done, PTAL

**Review by cybette — APPROVED** [review]
LGTM, thanks!

File: README.md [hunk 4]
```diff
@@ -19,17 +19,21 @@ This repository serves as a space for the Ansible Community Working Group to man
 
 ## Infrastructure and processes we build and/or support
 
+* [Ansible Community Forum](https://forum.ansible.com/): Official Ansible Discource forum.
```
Comment (GregSutcliffe) [inline README.md:22]: typo, DiscourSe

**Review by GregSutcliffe — CHANGES_REQUESTED** [review]
Found a typo :rofl:

File: README.md (hunk 4 above)
Comment (Andersson007) [reply README.md]: yeah, i know though sometimes strange things happen to my not native brain:)

Comment (Andersson007) [reply README.md]: fixed, thanks, merging

**Comment by Andersson007** [comment]
@cybette @GregSutcliffe thanks for reviewing!

**Pull request merged**